addOns/scripts/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Added
 - Help explaining how to interact with Automation Framework plans.
+- Support for relative file paths and ones including vars
```

Comment:
Add a period? (Likely needed in all the CHANGELOGs)

Also is there an issue that should be referenced in these?